dotCMS/src/main/java/com/dotcms/rest/api/v1/user/UserResource.java

```diff
@@ -430,22 +430,14 @@ public final Response logoutAs(@Context final HttpServletRequest request) {
 	@JSONP
 	@NoCache
 	@Produces({ MediaType.APPLICATION_JSON, "application/javascript" })
-	public final Response loginAsData(@Context final HttpServletRequest request) {
+	public final Response loginAsData(@Context final HttpServletRequest request, @QueryParam("filter") String filter,
+			@QueryParam("includeUsersNumber") boolean includeUsersNumber) {
```

Comment:
I think the variable name should be "includeUserCount" instead. The same applies to all its occurrences.